Add instructions to README for Managed HSM (#15720)
Azure/azure-sdk-for-net: sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

# Azure Key Vault key client library for .NET
Azure Key Vault is a cloud service that provides secure storage of keys for encrypting your data. 
Multiple keys, and multiple versions of the same key, can be kept in the Azure Key Vault. 
Cryptographic keys in Azure Key Vault are represented as [JSON Web Key (JWK)][JWK] objects.

> Azure Key Vault Managed HSM is a fully managed, highly available, single-tenant, standards-compliant cloud service that enables 

Reviewer: ```suggestion
Azure Key Vault Managed HSM is a fully-managed, highly-available, single-tenant, standards-compliant cloud service that enables 
```

> you to safeguards cryptographic keys for your cloud applications, using FIPS 140-2 Level 3 validated HSMs.

Reviewer: ```suggestion
you to safeguard cryptographic keys for your cloud applications using FIPS 140-2 Level 3 validated HSMs.
```

    az keyvault show --name <your-key-vault-name>
    ```

* Create the Azure Key Vault or Managed HSM and grant the above mentioned application authorization to perform administrative operations on the Managed HSM 

> (replace `<your-resource-group-name>` and `<your-key-vault-name>` with your own, unique names and `<your-service-principal-object-id>` with the value from above):

Reviewer: ```suggestion
(replace `<your-resource-group-name>` and `<your-key-vault-name>` with your own unique names and `<your-service-principal-object-id>` with the value from above):
```

> If you are creating a standard Key Vault resource, use the following CLI command. 

Reviewer: ```suggestion
If you are creating a standard Key Vault resource, use the following CLI command:
```

```PowerShell
az keyvault create --resource-group <your-resource-group-name> --name <your-key-vault-name>
```

> If you are creating a Managed HSM resource, use the following CLI command. 

Reviewer: ```suggestion
If you are creating a Managed HSM resource, use the following CLI command: 
```

```PowerShell
    az keyvault create --hsm-name <your-key-vault-name> --resource-group <your-resource-group-name> --administrators <your-service-principal-object-id> --location <your-azure-location>
```

#### Activate your managed HSM
This section only applies if you are creating a Managed HSM. All data plane commands are disabled until the HSM is activated. You will not be able to create keys or assign roles. 

> Only the designated administrators that were assigned during the create command can activate the HSM. To activate the HSM you must download the Security Domain.

Reviewer: ```suggestion
Only the designated administrators that were assigned during the create command can activate the HSM. To activate the HSM you must download the security domain.
```

Same as above. This isn't a product name, but used as a general term.

To activate your HSM you need:
- Minimum 3 RSA key-pairs (maximum 10)
- Specify minimum number of keys required to decrypt the security domain (quorum)

To activate the HSM you send at least 3 (maximum 10) RSA public keys to the HSM. The HSM encrypts the security domain with these keys and sends it back. 

> Once this security domain download is successfully completed, your HSM is ready to use. 

Reviewer: This reads weird. Maybe I missed it above, but I'd change it in both places:
```suggestion
Once this security domain is successfully downloaded, your HSM is ready to use. 
```

You also need to specify quorum, which is the minimum number of private keys required to decrypt the security domain.

The example below shows how to use openssl to generate 3 self signed certificate.

```PowerShell
openssl req -newkey rsa:2048 -nodes -keyout cert_0.key -x509 -days 365 -out cert_0.cer
openssl req -newkey rsa:2048 -nodes -keyout cert_1.key -x509 -days 365 -out cert_1.cer
openssl req -newkey rsa:2048 -nodes -keyout cert_2.key -x509 -days 365 -out cert_2.cer
```

Use the `az keyvault security-domain download` command to download the security domain and activate your managed HSM. 

> The example below, uses 3 RSA key pairs (only public keys are needed for this command) and sets the quorum to 2.

Reviewer: ```suggestion
The example below uses 3 RSA key pairs (only public keys are needed for this command) and sets the quorum to 2.
```
